### PR TITLE
Project Mongo API - Estefanía

### PR DESCRIPTION
**Pull request by esteficodes**
Link to deployed API: [https://bootcamps-app.herokuapp.com/bootcamps](url)

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,37 +1,125 @@
...
+})
+
+//To create the Bootcamp model
+const Bootcamp = mongoose.model('Bootcamp', bootcampSchema)
+
+//Seeding the database. Will only run if
+// RESET_DB environment variable is present and is true
+if (process.env.RESET_DB) {
+  const seedDatabase = async () => {
+   await Bootcamp.deleteMany()
```
Comment (Irina-web-dev) [inline server.js:39]: Indentation could be improved here 🐱‍👤

File: server.js [hunk 2]
```diff
@@ -1,37 +1,125 @@
...
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
-app.use(bodyParser.json())
+app.use(express.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
   res.send('Hello world')
 })
```
Comment (Irina-web-dev) [inline server.js:65]: I am wondering if this endpoint is really needed 🤔

File: server.js [hunk 3]
```diff
@@ -1,37 +1,125 @@
...
+    })
+    res.json(bootcamps)
+  }
+
+  if (hasGenderPerspective) {
+    const bootcamps = await Bootcamp.find({
+      hasGenderPerspective: true
+    })
+    res.json(bootcamps)
+  }
```
Comment (Irina-web-dev) [inline server.js:90]: I can image how useful these query parameters would be on the frontend side. Good job!✨

**Review by Irina-web-dev — APPROVED** [review]
Hi Estefania! 🦊

It was very interesting to go through the database of  bootcamps you have created, it is great that you approached this task creatively  🐱‍🏍

You have a lot of comments and I understand that you keep them to be able to look at and understand better what have you done later on. I do the same 🙌

Your code is clean and you have created useful endpoints following RESTfull convention.

God job!💪🦊✨

File: server.js [hunk 4]
```diff
@@ -1,37 +1,125 @@
...
+  review: String,
+  courses: String,
+  subjects: String,
+  description: String,
+  cost: String,
+  duration: String,
+  isFlexible: Boolean,
+  guaranteesJob: Boolean ,
+  hasGenderPerspective: Boolean,
+  url: String
```
Comment (KarolCislowski) [inline server.js:29]: Nice Schema 👍🏻

File: server.js (hunk 3 above)
Comment (KarolCislowski) [inline server.js:90]: Great filtering 😸

File: server.js [hunk 5]
```diff
@@ -1,37 +1,125 @@
...
+  // Member.find().then(data => {
+    //res.json(data)
+  //})
 
+  //v3 - mongoose
+  //Member.find((err, data) => {
+   // res.json(data)
+  //})
+
+})
```
Comment (KarolCislowski) [inline server.js:122]: 🧹

**Review by KarolCislowski — APPROVED** [review]
Is it 🐦 ? Is it ✈? NOOO IT IS ESTEFANIA!! 🦸🏻‍♀️ 


image

**Pull request closed without merging**